src/VisualStudio/Core/Def/Implementation/NavigationBar/NavigationBarClient.cs

```diff
@@ -152,7 +152,7 @@ int IVsDropdownBarClient.GetComboTipText(int iCombo, out string pbstrText)
             int selectionIndex;
             var selectedItemPreviewText = string.Empty;
 
-            if (_dropdownBar.GetCurrentSelection(iCombo, out selectionIndex) == VSConstants.S_OK)
+            if (_dropdownBar.GetCurrentSelection(iCombo, out selectionIndex) == VSConstants.S_OK && selectionIndex >= 0)
```

Comment:
Doesn't that sound like a bug in GetCurrentSelection? Won't everyone whom calls this method need to handle this now?